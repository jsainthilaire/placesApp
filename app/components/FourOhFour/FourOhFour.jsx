import React from 'react'
import { centeredContainer, title, description } from 'sharedStyles/styles.css'

const FourOhFour = () => (
  <section className={centeredContainer}>
    <h1 className={title}>Not Found</h1>
    <p className={description}>The page you are looking for does not exist</p>
  </section>
)

export default FourOhFour
