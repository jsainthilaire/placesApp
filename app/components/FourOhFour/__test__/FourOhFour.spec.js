import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import FourOhFour from '../FourOhFour'

test('FourOhFour should render correctly', () => {
  const wrapper = shallow(<FourOhFour />)
  expect(toJson(wrapper)).toMatchSnapshot()
})
