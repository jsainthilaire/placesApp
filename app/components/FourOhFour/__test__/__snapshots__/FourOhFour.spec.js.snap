// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FourOhFour should render correctly 1`] = `
<section
  className="centeredContainer"
>
  <h1
    className="title"
  >
    Not Found
  </h1>
  <p
    className="description"
  >
    The page you are looking for does not exist
  </p>
</section>
`;
